doc/source/migrations/index.rst

```diff
@@ -16,17 +16,15 @@ Version ``v9.0``
 - To use ``check-licenses: true`` with the ``ansys/actions/build-wheelhouse`` action, use Python version 3.10 or higher.
 - Update your workflow to not use ``use-trusted-publisher: true`` with our pypi release actions.
 
-  .. warning::
-
```

Comment:
Why did we remove the warning @clatapie? This should continue to be there

Reply:
This was not supposed to be removed!